src/Microsoft.OData.Core/ODataLibraryCompatibility.cs

```diff
@@ -4,14 +4,15 @@
 // </copyright>
 //---------------------------------------------------------------------
 
+using System;
 using System.Diagnostics.CodeAnalysis;
 
 namespace Microsoft.OData
 {
     /// <summary>
     /// Library compatibility levels.
     /// </summary>
-    [SuppressMessage("Microsoft.Design", "CA1008:EnumsShouldHaveZeroValue")]

```

Comment:
Add `None = 0` (or `None = 1 << 0`) to cure this compiler warning

Reply:
I think when I added the Flags attribute to this enum I removed the compiler warning. Flag enums start from 1